src/app/technical-record/tech-rec-history/tech-rec-history.component.spec.ts

```diff
@@ -1,67 +1,44 @@
-import {MaterialModule} from '@app/material.module';
-import {SharedModule} from '@app/shared/shared.module';
-import {RouterTestingModule} from '@angular/router/testing';
-import {FontAwesomeModule} from '@fortawesome/angular-fontawesome';
-import {NgrxFormsModule} from 'ngrx-forms';
-import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
-import {VehicleSummaryComponent} from '@app/technical-record/vehicle-summary/vehicle-summary.component';
-import {TechnicalRecordComponent} from '@app/technical-record/technical-record.component';
-
-describe('VehicleSummaryComponent', () => {
-
-  let component: VehicleSummaryComponent;
-  let fixture: ComponentFixture<VehicleSummaryComponent>;
+import { async, ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
+import { TechRecordHelpersService } from '@app/technical-record/tech-record-helpers.service';
+import { SharedModule } from '@app/shared/shared.module';
+import { RouterTestingModule } from '@angular/router/testing';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { TechRecHistoryComponent } from '@app/technical-record/tech-rec-history/tech-rec-history.component';
+import { Router } from '@angular/router';
+
+describe('TechRecHistoryComponent', () => {
+  let component: TechRecHistoryComponent;
+  let fixture: ComponentFixture<TechRecHistoryComponent>;
   let injector: TestBed;
+  let mockRouter = {
+    navigate: jasmine.createSpy('navigate')
+  };
 
   beforeEach(() => {
-
     TestBed.configureTestingModule({
-      imports: [
-        StoreModule.forRoot(appReducers),
-        HttpClientTestingModule,
-        MatDialogModule,
-        FormsModule,
-        ReactiveFormsModule,
-        BrowserAnimationsModule,
-        MaterialModule,
-        SharedModule,
-        RouterTestingModule,
-        FontAwesomeModule,
-        ReactiveFormsModule,
-        NgrxFormsModule
-      ],
-      declarations: [VehicleSummaryComponent, TechnicalRecordComponent],
-      providers: [
-        TechRecordHelpersService
-      ],
+      imports: [SharedModule, RouterTestingModule],
+      declarations: [TechRecHistoryComponent],
+      providers: [TechRecordHelpersService, { provide: Router, useValue: mockRouter }],
       schemas: [CUSTOM_ELEMENTS_SCHEMA]
-
     }).compileComponents();
 
-    fixture = TestBed.createComponent(VehicleSummaryComponent);
+    fixture = TestBed.createComponent(TechRecHistoryComponent);
     injector = getTestBed();
+    mockRouter = TestBed.get(Router);
     component = fixture.componentInstance;
-    component.activeRecord = {
-      'vin': 'XMGDE02FS0H012345',
-      'vehicleSize': 'small',
-      'testStationName': 'Rowe, Wunsch and Wisoky',
-      'vehicleId': 'JY58FPP',
-      'vehicleType': 'psv',
-      'axles': [
-        { 'parkingBrakeMrk': false, 'axleNumber': 1 },
-        { 'parkingBrakeMrk': true, 'axleNumber': 2 },
-        { 'parkingBrakeMrk': false, 'axleNumber': 3 }
-      ]
-    };
-
+    component.techRecordsJson = [
```

Comment:
This variable name is a bit strange, i  assume this is for debug purpose and that it is a recordObject?

Reply:
As agreed wit the team, code was delivered prior to ADR and reference ticket for future refactor if needed

Reference: https://jira.dvsacloud.uk/browse/CVSB-15101